app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -987,6 +990,7 @@ class AdvancedFilter extends React.Component {
             </React.Fragment>
           )}
         </Form.Group>
+        {numberOption !== 'between' && numberOption !== '' && this.renderStatementTooltip(filter.name, index, numberOptionTooltipText)}
         {numberOption === 'between' && this.renderStatementTooltip(filter.name, index, betweenTooltipText)}
       </React.Fragment>
     );
```

Comment:
should there be a tooltip for the address combo @hackrm? to explain the searching blank results

Reply:
Sure, I think including a tooltip for address referencing the ability to search for monitorees with a blank address would be beneficial. How about something like:

> Leaving this field blank will return monitorees with blank address fields.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1252/commits/1f615027510e14bc63773a1044e97dcb3f2af524

Reply:
I modified the language a bit since it wouldn't be specific to the address adv filter

Reply:
see @hackrm 's comment above

Reply:
Sorry, I will update the Jira ticket to reflect this

Reply:
no worries, sorry I probably read it before as the number option tooltip

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1252/commits/00b46c7e5d22ccb24f7f0a01ec951b89f46bca12